js/bybit.js

```diff
@@ -665,6 +668,29 @@ module.exports = class bybit extends Exchange {
         return this.milliseconds () - this.options['timeDifference'];
     }
 
+    handleOptionAndParamsValue (params, methodName, optionName, defaultValue = undefined) {
+        // This method can be used to obtain method specific properties, i.e: this.handleOptionAndParams (params, 'fetchPosition', 'marginMode', 'isolated')
+        const defaultOptionName = 'default' + this.capitalize (optionName); // we also need to check the 'defaultXyzWhatever'
```

Comment:
isn't the entire goal of overriding this here to remove this `default` thing? so users would have `enableUnifiedMargin` in options instead of `defaultUnifiedMargin` ?